ssl/ssl3.h

```diff
@@ -627,6 +627,9 @@ typedef struct ssl3_state_st {
      */
     unsigned char *alpn_selected;
     unsigned alpn_selected_len;
+    unsigned char *alpn_proposed;
```

Comment:
We don’t care about 16bit.  You'll have to pack things manually.  Or, use the SSLCERT thing like @ekasper said.